src/main/scala/viper/silver/reporter/Message.scala

```diff
@@ -269,11 +270,31 @@ case class CopyrightReport(override val text: String) extends SimpleMessage(text
 }
 
 case class MissingDependencyReport(override val text: String) extends SimpleMessage(text) {
-  override val name = "missing_dependency_report"
+  override val name: String = "missing_dependency_report"
 }
 
 // FIXME: for debug purposes only: a pong message can be reported to indicate
 // FIXME: that the verification backend is alive.
 case class PongMessage(override val text: String) extends SimpleMessage(text) {
   override val name: String = "dbg__pong"
 }
+
+case class QuantifierInstantiationsMessage(quantifier: String, instantiations: Int,
```

Comment:
Add a comment clarifying what `quantifier` is supposed to be: a string representation of a quantified formula? The Viper name or SMT QID identifying a quantified expression? One of the strings `"forall"` or `"exists"`?